app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    for one in people:
+        Person(one["name"], one["age"])
+
+    for one in people:
+        if "wife" in one and one["wife"] is not None:
+            Person.people[one["name"]].wife = Person.people[one["wife"]]
+
+        elif "husband" in one and one["husband"] is not None:
+            Person.people[one["name"]].husband = Person.people[one["husband"]]
```

Comment:
You could retrieve the `Person` instance at the beginning of an iteration instead of calling it twice in if condition:
```suggestion
    for one in people:
        person_instance =  Person.people[one["name"]]
        if "wife" in one and one["wife"] is not None:
            person_instance.wife = Person.people[one["wife"]]

        elif "husband" in one and one["husband"] is not None:
            person_instance.husband = Person.people[one["husband"]]
```